src/igvfd/searches/configs/matrix_file.py

```diff
@@ -6,6 +6,67 @@
 )
 def matrix_file():
     return {
+        'facets': {
+            'uuid': {
+                'title': 'UUID'
```

Comment:
UUID and accession don't seem to be included in facets for other File subclasses except Signal File. I'm confused if we should be adding UUID and accession to every subclass, or if we should be omitting them?

Reply:
Yes I coped these from signal file, thanks for catching that I don't see the use in having uuid and accession in the facets. It looks like Mingjie had added them there but I don't think they serve any purpose on the search page so I will remove them.